src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -163,15 +168,20 @@ export const getBookInformation = async(
     return [{id, bookVersion}];
   };
 
-  const allReferences = reference.bookId ? getInputReferenceInfo(reference.bookId, reference.bookVersion)
+  const allReferences = reference.bookId ? await getInputReferenceInfo(reference.bookId, reference.bookVersion)
     : await services.archiveLoader.getBookIdsForPage(reference.pageId);
 
-  const configuredReference = allReferences.filter((item) => BOOKS[item.id])[0];
+  const configuredReference = allReferences.filter(async(item) =>
+    await services.bookConfigLoader.getBookVersionFromUUID(item.id))[0];
 
   if (configuredReference) {
     const osWebBook =  await services.osWebLoader.getBookFromId(configuredReference.id);
     const archiveBook = await services.archiveLoader.book(
-      configuredReference.id, BOOKS[configuredReference.id].defaultVersion
+      configuredReference.id,
+      assertDefined(
```

Comment:
i think the reference has a version on it, you might not have to do this here

Reply:
this is added in https://github.com/openstax/rex-web/pull/1133

Reply:
Should I wait for #1133 to be merged? @PiotrKozlowski @TomWoodward

Reply:
You could either merge my branch into yours and change the base for this PR or you could just wait and resolve conflicts when my PR is merged - idk when this will happen